src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/security.yml

```diff
@@ -85,7 +85,7 @@ services:
 
     # Session
     self_service.security.authentication.session.session_storage:
-        public: false
+        public: true
```

Comment:
This certainly is the easiest way to achieve this, but consider using DI to get this service into the controller. That way this is more future proof.

Reply:
Agree, but at this point since the changeset is already so big i do not want to change too much. Especially not refactorings that should go in a different PR. The setting was wrong before. Apparently SF did not do much with the public directive.